FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -22,13 +22,13 @@
 #import <FirebaseCore/FIRLogger.h>
 #import <FirebaseCore/FIROptionsInternal.h>
 #import "FirebaseRemoteConfig/Sources/FIRRemoteConfigComponent.h"
-#import "FirebaseRemoteConfig/Sources/FIRRemoteConfig_Internal.h"
+#import "FirebaseRemoteConfig/Sources/Private/FIRRemoteConfig_Internal.h"
```

Comment:
Would be nice to rename to FIRRemoteConfig_Private.h

Reply:
Done.